guides/common/modules/proc_enabling-red-hat-repositories.adoc

```diff
@@ -8,11 +8,6 @@ To select the repositories to synchronize, you must first identify the product t
 For {RHEL} 8, you must enable both AppStream and BaseOS repositories.
 
 ifdef::satellite[]
-.Disconnected {Project}
-If you use Disconnected {ProjectServer}, you must configure {Project} to synchronize content with a local CDN server before synchronizing content.
-For more information, see xref:Configuring_Server_to_Synchronize_Content_with_a_Local_CDN_Server_{context}[].
-endif::[]
-
```

Comment:
Do we want to include a different text instead of this with the new ISS workflow?